fixes.inc

```diff
@@ -700,6 +700,20 @@ in the output code.
  *     <author    href="https://github.com/Y-Less/" >Y_Less</author>
  * </fix>
  *
+ * <fix name="GetVehicleComponentSlot">
+ *     <problem>
+ *         	The function GetVehicleComponentInSlot doesn't work for CARMODTYPE_STEREO.
+ *			Both front bull bars and front bumper components are saved in the CARMODTYPE_FRONT_BUMPER slot. If a vehicle has both of them installed, this function will only return the one which was installed last.
```

Comment:
Please try limit line lengths, but this is a very minor point.

Reply:
Done in https://github.com/pawn-lang/sa-mp-fixes/pull/149/commits/8f85355c1cb216367e9b7b38783f0f31068372d3.